app/models/server/models/Users.js

```diff
@@ -135,7 +135,11 @@ export class Users extends Base {
 		return this.findOne(query);
 	}
 
-	findOneOnlineAgentByUsername(username, options) { // TODO:: Create class Agent
+	findOneOnlineAgentByUsername(userList, options) { // TODO:: Create class Agent
```

Comment:
should we rename or create a new method? "by username" doesn’t look complete compatible with 'user~~names~~List'